cabal-install/Distribution/Client/Install.hs

```diff
@@ -1450,19 +1452,31 @@ installUnpackedPackage verbosity buildLimit installLock numJobs
             withWin32SelfUpgrade verbosity ipid configFlags
                                  cinfo platform pkg $ do
               setup Cabal.copyCommand copyFlags mLogPath
-              when shouldRegister $ do
-                setup Cabal.registerCommand registerFlags mLogPath
 
             -- Capture installed package configuration file, so that
             -- it can be incorporated into the final InstallPlan
-            maybePkgConfs <- maybeGenPkgConfs mLogPath
-
-            return (Right (BuildOk docsResult testsResult maybePkgConfs))
+            -- TODO: This is duplicated with
```

Comment:
I couldn't find the fragment you're referring to, would be nice if this comment was a bit more expanded.